src/renderer/views/Watch/Watch.js

```diff
@@ -1137,6 +1141,20 @@ export default Vue.extend({
       }, 100)
     },
 
+    GoToNextVideo: function () {
```

Comment:
Please make the first letter of the function name lowercase.

Reply:
Fixed now